docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "2"
+services:
+  db:
+    image: postgres
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+  pulse:
+    image: rabbitmq:3-management
```

Comment:
Standard vs alpine is 76MB vs 44MB:
https://hub.docker.com/r/library/rabbitmq/tags/